templates/set_safe_threshold.cdc

```diff
@@ -0,0 +1,22 @@
+
+   
+import FlowDKG from 0x8624b52f9ddcd04a
+
+transaction(newThresholdPercentage: UFix64?) {
```

Comment:
```suggestion
transaction(newThresholdPercentage: UFix64) {
```

I got an error when passing a `UFix64` type argument when this accepted a `UFix64?` parameter using Flow CLI `--arg` flag, guessing the same would be true with `--args-json`